static/js/answers-experience-frame.js

```diff
@@ -4,9 +4,11 @@ export default class AnswersExperienceFrame {
   constructor (runtimeConfig) {
     this.runtimeConfig = runtimeConfig;
     this._hasManuallyInitialized = false;
-
-    runtimeConfig._onUpdate(updatedConfig => {
-      sendToIframe({ runtimeConfig: updatedConfig });
+    
+    Object.entries(runtimeConfig.getAll()).forEach(([key, value]) => {
+      runtimeConfig._onUpdate(key, configOption => {
+        sendToIframe({ runtimeConfig: {key: configOption} });
+      });
```

Comment:
When someone sets a new `runtimeConfig` options that wasn't set when `AnswersExperienceFrame` was constructed, I don't think it will be sent to the iframe because it was never registered to listen to that key. So I think it would be best if we make the key-specific listeners an additive change.

Reply:
right. I should update runtime-config.js to support both `generalUpdateListener` and `keySpecificListeners`. AnswersExperienceFrame would need a generalUpdateListener to catch any types of key update to send to iframe, and AnswersExperience will need keySpecificListeners for handling specific key set from runtime config receiver. That sounds ok?

Reply:
Ya, that sounds good!